### ISSUE TITLE
Pod Template issue with WPML enabled sites

### ISSUE THREAD

**Issue by Tom2527** [pods-framework/pods#3242]
I have a Wordpress with both the Pods plugin and WPML plugin as I need the site to be both in English and Dutch. I checked the template option "Make 'Pod Templates' translatable" and I created a Pods Template that I translated in both languages. 

This template is shown AFTER a specific Pod is shown. Everything works fine, except that the English template is shown at all times. Never the Dutch one. 

If I rename the English template to "mytemplate_ENG" only the Dutch template is shown at all times. Both translations are linked to each other using the WPML interface.

<img width="814" alt="screen shot 2015-11-24 at 13 24 17" src="https://cloud.githubusercontent.com/assets/15998590/11366659/f1c8991a-92ae-11e5-9339-25fa790ea053.png">
<img width="1110" alt="screen_shot_2015-11-24_at_13_25_01" src="https://cloud.githubusercontent.com/assets/15998590/11366660/f1edf3d6-92ae-11e5-9bf8-19664a7026af.png">

**Comment by jimtrue** [comment]
This is from this Forum Post: http://pods.io/forums/topic/wpml-and-pods-templates/

It looks like to me that the Auto Template Options isn't recognizing the translation settings/branch coming from the Add/Edit Pod Template Editor. Since a branch is created (using the same name for the Template) with the multiple translations, it's just not picking up that choice in the Auto Template Options based on the language selected. Going to have to push this one up the chain of command for troubleshooting. @quasel or @Ramoonus, do you guys have any experience with WPML and can add any pointers?

**Comment by jamesgol** [comment]
I'm still going to dig into this a bit more to come up with the best solution but here is a simple fix that should not cause any issues (because it uses a filter).  Some of the other translation plugins might have the same issue
In includes/general.php:pods_by_title()
Change:
```
    if ( $page )
        return get_post( $page,  $output );
```
to
```
    if ( $page )
        return get_post( apply_filters( 'wpml_object_id', $page, $type, true ), $output );
```

**Comment by jimtrue** [comment]
@jamesgol the forum poster I referenced for this issue tried your solution and it worked perfectly. Wanted to let you know ;)

**Comment by JoryHogeveen** [comment]
Wasn't this issue fixed/closed with #3537 ?
